octobot_evaluators/evaluator/strategy_evaluator.py

```diff
@@ -195,17 +195,22 @@ def _get_available_node_paths(self,
                                   use_cache=True):
         if use_cache:
             try:
-                return self.available_node_paths_cache[matrix_id][exchange_name][evaluator_type]
+                return self.available_node_paths_cache[matrix_id][exchange_name][evaluator_type][cryptocurrency][symbol]
```

Comment:
Why this was not already with `[cryptocurrency][symbol]` ?

Reply:
No idea but it was a bug

Reply:
ok :+1: